nestkernel/nest_types.h

```diff
@@ -57,7 +57,7 @@
  * convenient interface to the NEST simulator.  Front. Neuroinform. 2
  * : 12. doi:10.3389/neuro.11.012.2008
  *
- * (C) Copyright 1995-2006 The NEST Initiative.
+ * Copyright (C) 2004 The NEST Initiative.
```

Comment:
This copyright notice should disappear entirely, the copyright at the top of the file suffices.